scripts/wawCommons.py

```diff
@@ -257,19 +257,19 @@ def _fileMatchesPatterns(filename, patterns):
     return False
 
 def getWorkspaceId(config, workspacesUrl, version, username, password):
-    if hasattr(config, 'conversation_workspace_id') and getattr(config, 'conversation_workspace_id'):
-        logger.info('conversation_workspace_id defined.')
-        workspaceId = getattr(config, 'conversation_workspace_id')
-    else:
-        logger.info('conversation_workspace_id parameter not defined.')
+
+    workspaceId = getOptionalParameter(config, 'conversation_workspace_id')
+
+    if not workspaceId:
         workspaceId = ""
 
-        # workspace name unique
-        if hasattr(config, 'conversation_workspace_name_unique') and getattr(config, 'conversation_workspace_name_unique') in ["true", "True"]:
-            if hasattr(config, 'conversation_workspace_name') and getattr(config, 'conversation_workspace_name'):
-                logger.info('conversation_workspace_name set to unique')
-                workspaceName = getattr(config, 'conversation_workspace_name')
+        # workspace name is considered as unique
+        workspaceNameUnique = getOptionalParameter(config, 'conversation_workspace_name_unique')
```

Comment:
I would just suggest to change name of this param to something like: `conversation_workspace_deploy_by_name`, it is the real action we tried to do and we test if the name is unique... when we change it... we just need to slightly change log output and it will make more sense:
- 'conversation_workspace_name set to unique' -> something like 'conversation_workspace_deploy_by_name is set to true -> trying deploy workspace by name'
- "'conversation_workspace_name' set to unique but not defined." -> 'conversation_workspace_deploy_by_name is set to true but no workspace name is provided (conversation_workspace_name)'

Reply:
Please create new issue for this.

Reply:
#289